scim/resource_user_test.go

```diff
@@ -585,11 +590,86 @@ func TestResourceUserDelete_NonExistingDir(t *testing.T) {
 		ID:       "abc",
 		HCL: `
 			user_name    = "abc"
+			disable_as_user_deletion = false
 			force_delete_home_dir = true
 		`,
 	}.Apply(t)
 	assert.EqualError(t, err, "force_delete_home_dir: Path (/Users/abc) doesn't exist.")
 }
+func TestResourceUserDeleteAsDisable(t *testing.T) {
+	qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "PATCH",
+				Resource: "/api/2.0/preview/scim/v2/Users/abc",
+			},
+		},
+		Resource: ResourceUser(),
+		Delete:   true,
+		ID:       "abc",
+		HCL: `
+			user_name    = "abc"
+			disable_as_user_deletion = true
+		`,
+	}.ApplyNoError(t)
+}
+
+func TestResourceUserDeleteAsDisableWithForceDeleteRepos(t *testing.T) {
+	_, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "PATCH",
+				Resource: "/api/2.0/preview/scim/v2/Users/abc",
```

Comment:
can you assert for body with `ExpectedRequest`? for this new functionality it's critical to assert on the body.

Reply:
Done